stable/helm-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "0.1.0"
+appVersion: "0.2.0"
```

Comment:
app version would be 0.3.0

Reply:
@sstarcher Done, thanks for pointing out. Still trying to get `helm ct` to work so that I don't miss this out in the future.

Reply:
@naiduarvind what's `helm ct`